tensorflow/python/compiler/tensorrt/test/tf_trt_integration_test_base.py

```diff
@@ -1027,15 +1030,18 @@ def RunTest(self, run_params):
         current_input_data = []
         for spec, np_shape in zip(input_specs, dim_list):
           np_dtype = spec.dtype.as_numpy_dtype()
-          # Multiply the input by some constant to avoid all zeros input for
-          # integer types.
-          scale = 10.0 if np.issubdtype(np_dtype, np.integer) else 1.0
-          # TODO(laigd): add debug options. E.g. we can set the input data to be
-          # continuous natural numbers:
-          # seq = np.arange(np.prod(np_shape))
-          # seq.resize(np_shape)
-          # current_inputs_data.append(scale * seq.astype(np_dtype))
-          data = (scale * np.random.random_sample(np_shape)).astype(np_dtype)
+          if not np.issubdtype(np_dtype, np.bool_):
+            # Multiply the input by some constant to avoid all zeros input for
+            # integer types.
+            scale = 10.0 if np.issubdtype(np_dtype, np.integer) else 1.0
+            # TODO(laigd): add debug options. E.g. we can set the input data
+            #  to be continuous natural numbers:
+            # seq = np.arange(np.prod(np_shape))
+            # seq.resize(np_shape)
+            # current_inputs_data.append(scale * seq.astype(np_dtype))
```

Comment:
Does this TODO still make sense? we should update the TODO to reflect the code changes, if we really want to keep this TODO. Otherwise, just remove it.

Reply:
DONE.